### DISCUSSION TITLE
Does anyone have any idea how to use DepthOfFieldBlurPostProcess?

### DISCUSSION THREAD

**Discussion by jgabriel2302** [brianzinn/react-babylonjs#322] [Q&A]
I've tried several times and none of them work. Can anyone give me some advice?

**Comment by brianzinn** [comment]
what did you try? looks like you need to get the camera and pass it.
https://doc.babylonjs.com/typedoc/classes/BABYLON.DepthOfFieldBlurPostProcess#constructor
